dom/src/utils.ts

```diff
@@ -73,11 +73,11 @@ export function isEqualNamespace(
 }
 
 export function makeInsert(
-  map: Map<string, Map<Element, any>>,
+  map: Map<string, Map<Element, any>>
 ): (type: string, elm: Element, value: any) => void {
   return (type, elm, value) => {
     if (map.has(type)) {
-      const innerMap = map.get(type) as Map<Element, any>;
+      const innerMap = map.get(type)!;
```

Comment:
What's this? :eyes:

Reply:
a non-null assertion. `get` returns `x | undefined`, this is to force the result to be just `x`. The previous did the same, but for some reason tslint complains that it does not change the type of the expression.

Reply:
wow, this is really useful, is it how new in TypeScript?

Reply:
since TypeScript 2.0 apparently